docs/features/navigation.md

```diff
@@ -0,0 +1,121 @@
+# WooCommerce Navigation
+
+The WooCommerce Navigation feature is a navigational project designed to create a more intuitive and functional WooCommerce specific navigation.
+
+This API will allow you to add in your own items to the navigation and register pages with the new navigation screens.
+
+### Getting started
+
+This feature is hidden behind a feature flag and can be turned on or off by visiting WooCommerce -> Settngs -> Advanced -> Features and checking the box next to the `Navigation` option.  It can also by controlled programmatically by setting the option `woocommerce_navigation_enable` to `yes` or `no`. 
+
+The fastest way to get started is by creating an example plugin from WooCommerce Admin.  Inside your `woocommerce-admin` directory, enter the following command:
+
+`npm run example -- --ext=add-navigation-items`
+
+This will create a new plugin that covers various features of the navigation and helps to register some intial items and categories within the new navigation menu.
+
```

Comment:
I've seen devs get confused by the magic of webpack copying files to the actual plugin. Being able to edit directly in wc-admin may not be intuitive, so a line to the effect of "you can edit directly in wc-amin" pointing to the source files https://github.com/woocommerce/woocommerce-admin/tree/main/docs/examples/extensions/add-navigation-items could be useful.

Reply:
Great point.  Added a comment about how this works in https://github.com/woocommerce/woocommerce-admin/pull/5537/commits/7d98b9f3040bfd11d95212cf84fa76b6a7394d60